lib/brexit_checker/action.rb

```diff
@@ -23,7 +23,6 @@ def show?(selected_criteria)
 
   def self.load(params)
```

Comment:
Can we just call `new` now and lose this method?

Reply:
Ah yeah - good shout!